src/debug/debug.proto

```diff
@@ -23,6 +23,7 @@ message Filter {
     bool pachd = 1;	
     pps_v2.Pipeline pipeline = 2;	
     Worker worker = 3;
+    bool postgres = 4;
```

Comment:
minor suggestion: should we name this `database`? Even though we are tightly coupled to postgres, the API doesn't necessarily need to acknowledge that

Reply:
Sure, that makes sense!